src/ray/raylet/worker_pool_test.cc

```diff
@@ -26,7 +26,7 @@ namespace ray {
 
 namespace raylet {
 
-int MAXIMUM_STARTUP_CONCURRENCY = 5;
+int MAXIMUM_STARTUP_CONCURRENCY = 15;
```

Comment:
Why change this?

Reply:
Because there is a case holding more than 5 pending registration workers. Before this PR, the number of worker slot is 15(`MAXIMUM_STARTUP_CONCURRENCY` * `NUM_WORKER_PER_PROCESS`).